### PR TITLE
fix invisible text

### PR DESCRIPTION
**Pull request by imor**
## What kind of change does this PR introduce?

docs update

## What is the current behavior?

While following the NextJS getting started guide the http://localhost:3000/countries page displays black text on black background. This confused me as I thought the page was blank.

## What is the new behavior?

The text shown on this page will be white on a black background.

## Additional context

N/A

### CODE REVIEW

**Review by kiwicopple — APPROVED** [review]

File: apps/docs/pages/guides/getting-started/quickstarts/nextjs.mdx [hunk 1]
```diff
@@ -100,7 +100,7 @@ export const meta = {
           const { data: countries } = await supabase.from("countries").select();
 
           return (
-            <ul className="my-auto">
+            <ul className="my-auto text-white">
```
Comment (kiwicopple) [inline apps/docs/pages/guides/getting-started/quickstarts/nextjs.mdx:103]: I don't know if this will work on light mode?

Comment (kiwicopple) [reply apps/docs/pages/guides/getting-started/quickstarts/nextjs.mdx:103]: I think there is some sort of `brand-` class we can use insted

**Review by kiwicopple — COMMENTED** [review]
see comments inline

**Review by kiwicopple — APPROVED** [review]

File: apps/docs/pages/guides/getting-started/quickstarts/nextjs.mdx (hunk 1 above)
Comment (imor) [reply apps/docs/pages/guides/getting-started/quickstarts/nextjs.mdx]: You were right @kiwicopple, `text-white` would have displayed white text on white background in light mode. The correct class is `text-foreground`. The `foreground` and `background` colours are defined in the `tailwind.config.js` file. Now it works for both light and dark modes.

File: apps/docs/pages/guides/getting-started/quickstarts/nextjs.mdx [hunk 2]
```diff
@@ -100,7 +100,7 @@ export const meta = {
           const { data: countries } = await supabase.from("countries").select();
 
           return (
-            <ul className="my-auto">
+            <ul className="my-auto text-foreground">
```
Comment (MildTomato) [inline apps/docs/pages/guides/getting-started/quickstarts/nextjs.mdx:103]:
```suggestion
<ul className="my-auto text">
```

shorthand tailwind

Comment (MildTomato) [reply apps/docs/pages/guides/getting-started/quickstarts/nextjs.mdx:103]: oh wait, sorry, i misunderstood the content on the page

Comment (MildTomato) [reply apps/docs/pages/guides/getting-started/quickstarts/nextjs.mdx:103]: sorry, your right @imor

**Pull request merged**